Casks/n/netnewswire.rb

```diff
@@ -35,11 +35,16 @@
 
   zap trash: [
     "~/Library/Application Scripts/com.ranchero.NetNewsWire-Evergreen",
+    "~/Library/Application Scripts/com.ranchero.NetNewsWire-Evergreen.Mac.ShareExtension",
     "~/Library/Application Scripts/com.ranchero.NetNewsWire-Evergreen.Subscribe-to-Feed",
+    "~/Library/Application Scripts/com.ranchero.NetNewsWire-Evergreen.SubscribeToFeed",
```

Comment:
```suggestion
    "~/Library/Application Scripts/com.ranchero.NetNewsWire-Evergreen*",
```

We could combine those entries using a glob `*`.